test/teststl.cpp

```diff
@@ -1918,6 +1918,18 @@ class TestStl : public TestFixture {
                       "[test.cpp:7]: (error) Dereference of an invalid iterator: f().end()+1\n",
                       errout.str());
 
+        check("std::vector<int>& f();\n"
+              "std::vector<int>& g();\n"
```

Comment:
`g()` is not used in this test.

Reply:
Copy-paste. I fixed it also in original test case